datafusion/substrait/src/logical_plan/consumer.rs

```diff
@@ -972,7 +944,7 @@ pub async fn from_substrait_rex_vec(
 }
 
 /// Convert Substrait FunctionArguments to DataFusion Exprs
-pub async fn from_substriat_func_args(
+pub async fn from_substrait_func_args(
```

Comment:
fix'd the spelling - this would break consumer if someone is using this function, but I dunno why anyone would be. Still it's not necessary change so I'm happy to revert if that'd be better

Reply:
I think it is a nice improvement